libvirt/README.md

```diff
@@ -0,0 +1,53 @@
+# Machine Stats for Virtual environments (Virt-Stats)
+
+A simple and effective way to gather guest VM statistics (hostname, IP addresses) from a [libvirt](https://libvirt.org/)-based environment as a first layer of a [Tidal Migrations discovery process](https://guides.tidalmg.com/).
+
+## Prerequisutes
```

Comment:
I think we should include `Python 3` as a dependency as well. Do we need a minimal version? like 3.8? What do you think?